README.md

```diff
@@ -1,48 +1,58 @@
-# JSONForms - More Forms. Less Code
+# JSON Forms - More Forms. Less Code
+
 ### Complex Forms in the blink of an eye
```

Comment:
We should not use headers for this tagline. I would prefer emphasizing it, for example `*Complex forms in the blink of an eye*`

Bonus points if there is no markdown linter warning